charts/studio/templates/persistentvolume.yaml

```diff
@@ -8,7 +8,9 @@ spec:
   {{- with .Values.global.blobvault.persistentVolume.accessModes }}
   {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- if .Values.global.blobvault.persistentVolume.storageClassName }}
   storageClassName: {{ .Values.global.blobvault.persistentVolume.storageClassName }}
```

Comment:
@jesper7 Let's ignore the failed workflow. We will see it after the merge and fix it. 

I think worklow fails because it uses previous chart, which had enforced StorageClassName